tests/browser-tests.js

```diff
@@ -0,0 +1,61 @@
+/* globals describe, before */
+
+// const expect = require('chai').expect
+const async = require('async')
+const store = require('idb-blob-store')
+const tests = require('./repo-test')
+const _ = require('lodash')
+const IPFSRepo = require('../src')
+const idb = require('idb-blob-store')
+
+global.Buffer = require('buffer').Buffer
+
+var repoContext = require.context('raw!./test-repo', true)
+
+describe('IPFS Repo Testson on the Browser', function () {
+  this.timeout(10000)
+
+  before(function (done) {
+    var repoData = []
+    repoContext.keys().forEach(function (key) {
+      repoData.push({
+        key: key.replace('./', ''),
+        value: repoContext(key)
+      })
+    })
+
+    var mainBlob = store('ipfs')
+    var blocksBlob = store('ipfs/blocks/')
+
+    async.eachSeries(repoData, (file, cb) => {
+      if (_.startsWith(file.key, 'datastore/')) {
+        return cb()
+      }
+
+      const blob = _.startsWith(file.key, 'blocks/')
+        ? blocksBlob
+        : mainBlob
+
+      blob.createWriteStream({
+        key: file.key
+      }).end(file.value, cb)
+    }, () => {
+      // FIXME debuging possible idb-blob-store bug
+      setTimeout(done, 1000)
```

Comment:
This line was added for testing purposes.

It seems that the `end` event on the returned write streams is emitted before it actually finishes, making the testes fail, since the db isn't consistent.